maintainers/scripts/update.py

```diff
@@ -12,27 +12,31 @@
 
 class CalledProcessError(Exception):
     process: asyncio.subprocess.Process
+    stderr: Optional[bytes]
 
 class UpdateFailedException(Exception):
     pass
 
 def eprint(*args, **kwargs):
     print(*args, file=sys.stderr, **kwargs)
 
-async def check_subprocess(*args, **kwargs):
+async def check_subprocess_output(*args, **kwargs):
```

Comment:
I would use `outputs` or `output_and_error` to distinguish it from `subprocess.check_output`, which might give people reading the code an incorrect expectation.

Reply:
Updated to `check_subprocess_outputs` :+1: